packages/preview-button/admin/src/components/PreviewLink/index.js

```diff
@@ -1,37 +1,180 @@
+import { Dialog, DialogBody, DialogFooter } from '@strapi/design-system';
+import { Button } from '@strapi/design-system/Button';
 import { LinkButton } from '@strapi/design-system/LinkButton';
-import { useCMEditViewDataManager } from '@strapi/helper-plugin';
+import { useCMEditViewDataManager, useFetchClient } from '@strapi/helper-plugin';
 import Eye from '@strapi/icons/Eye';
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import { renderToString } from 'react-dom/server';
+import { useIntl } from 'react-intl';
+import '@utrecht/component-library-css';
+import '@utrecht/design-tokens/dist/index.css';
+import '@utrecht/component-library-css/dist/html.css';
 import usePluginConfig from '../../hooks/use-plugin-config';
-
+import './index.css';
```

Comment:
Voor performance is het beter om imports met side-effects (zoals CSS injection) helemaal aan het eind te doen, zodat de andere imports allemaal parallel geladen kunnen worden.

Dus alle `import '*.css';` als laatste.

Ik dacht dat er ook een ESLint regel voor was om dit te herkennen, ergens in een plugin ofzo, maar ik kan die zo gauw niet vinden: https://github.com/import-js/eslint-plugin-import/blob/HEAD/docs/rules/order.md